### PR TITLE
[readme] Update README to better document build system.

### LINKED ISSUE #78: lablgtk3 beta6 build-depends on camlp4
Hi,

somewhere between beta3 and beta6 a build-dependency on camlp4 seems to have creeped in, or at least dune thinks so:
```
dune build
        bash src/propcc.ml (exit 127)
(cd _build/default/src && /bin/bash -e -u -o pipefail -c 'camlp4o pr_o.cmo -impl propcc.ml4 -o propcc.ml')
/bin/bash: camlp4o: command not found
        bash src/varcc.ml (exit 127)
(cd _build/default/src && /bin/bash -e -u -o pipefail -c 'camlp4o pr_o.cmo -impl varcc.ml4 -o varcc.ml')
/bin/bash: camlp4o: command not found
File "src/dune", line 11, characters 12-29:
11 |  (libraries dune.configurator))
                 ^^^^^^^^^^^^^^^^^
Error: Library "dune.configurator" not found.
Hint: try: dune external-lib-deps --missing @@default
```
This the release of lablgtk3 tagged  3.0.beta6.

-Ralf.

### PR DESCRIPTION
**Pull request by ejgallego**
Closes #78

### CODE REVIEW

**Comment by ejgallego** [comment]
Thanks @XVilka , indeed we will merge this once #77 is merged.

File: README.md [hunk 1]
```diff
@@ -39,70 +39,47 @@ Failure "gdk_pixbuf_get_file_info unsupported in Gtk 2.x < 2.4"
 
 ## How to compile:
 
-  Type
+  To compile in developer mode type:
 ```
         dune build
```
[the remark this answers is not in the document]
Comment (ejgallego) [reply README.md]: Done thanks!

**Comment by ejgallego** [comment]
I think this is good to go for now.

**Comment by ejgallego** [comment]
Ping @garrigue

**Comment by garrigue** [comment]
I merged, but you could have done it.

**Comment by ejgallego** [comment]
Thanks!

**Pull request merged**